frontend/src/components/login/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from "react";
+import { Container, Row } from "react-bootstrap";
+import "./login.css";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      username: "",
+      password: "",
+      errors: {
+        username: "",
+        password: ""
+      },
+      usernameValid: false,
+      passwordValid: false,
+      formValid: false
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange = event => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    // proceed to the backend to validate the admin user data
+    console.log("data is being sent", this.state);
+    // fetch("http://the backend", {
+    //   method: "POST",
+    //   body: JSON.stringify(this.state),
+    //   headers: { "Content-Type": "application/json" }
+    // }).then(function() {
+    //   console.log("data has been set");
+    // });
+
+    this.setState({
+      username: "",
+      password: ""
+    });
+  };
+
+  render() {
+    const { errors } = this.state;
```

Comment:
since we are going to handle errors from the BE -> lets just remove this for now